veevavault/1.3/modules/ROOT/pages/veevavault-connector-examples.adoc

```diff
@@ -78,51 +95,54 @@ payload.data map () -> {
     <error-handler >
-        <on-error-propagate enableNotifications="true" logException="true" doc:name="On Error Propagate" doc:id="19d98427-1566-4f82-b694-f59cc6eb169a" type="ANY" >
-            <logger level="INFO" doc:name="Logger" doc:id="9c23d079-fbf2-42e7-afe2-b5186a7b9a3f" message="Flow Exception Handled --  #[payload]" />
+        <on-error-propagate enableNotifications="true" logException="true" doc:name="On Error Propagate" type="ANY" >
+            <logger level="INFO" doc:name="Logger" message="Flow Exception Handled --  #[payload]" />
         </on-error-propagate>
     </error-handler>
 </flow>
 ----
 
+[[retrieve-document-details]]
 == Retrieve Document Details
 
-This example retrieves document details based on type, subtype, and classification and stores the document details in the database.
+This example retrieves document details based on document type, subtype, and classification and stores the document details in the database.
 
-To create the Mule flow:
+The following screenshot shows the Studio app flow for this example:
 
-. Drag and drop *HTTP Listener* from the Mule Palette to the project canvas.
+image::veevavault-connector-get-documents-example.png[Get Documents operation Mule flow]
+
+. From the *Mule Palette* view, select *HTTP* and drag the *Listener* source to the canvas.
 . Configure HTTP Listener for your environment.
-. Drag and drop the *Get Documents* operation from VeevaVault Connector palette.
-. Enter the values for the Veeva Vault connection: +
+. From the *Mule Palette* view, select *Veeva Vault* and drag the *Get Documents* operation.
+. Enter the following values for the Veeva Vault connection: +
 * Username
 * Password
 * Vault URL
 * Vault version
 * Vault client ID
 . Click the respective refresh button to fetch the document type, subtype, and classification list.
-. Select the type, subtype, and classification from each of the respective drop-down lists.
+. Select the document type, subtype, and classification from each of the respective drop-down lists.
 . Add the document properties in the Document Properties list for your business requirements to incorporate the properties name in the VQL query to build dynamically.
-. Drag the *For-Each* scope after the Veeva Vault *Get Documents* operation.
-. Drop the *Transform Message* component and Database Connector *Insert* operation onto the project from the respective palettes.
-. Configure the transformation according to your business requirements using Dataweave.
-. Configure the Database *Insert* query to insert data into table.
-. Drag the *Set Payload* transformer after *For-Each* to set a payload to return to the triggered source.
-. Drag the *Logger* component to log information in the console or log file.
-
-image::veevavault-connector-get-documents-example.png[]
+. From the *Mule Palette* view, select *Core* and drag the *For Each* scope containing the following elements:
```

Comment:
Same question regarding the components. I would think you would have to configure the Database connector.